### ISSUE TITLE
QA Ph1_Accessibility: User cannot play video with keyboard only 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#206]
Using the keyboard only, the user is unable to play the video once it has been selected. 

Steps to reproduce: 
1. Go to https://phase1-dot-io-webapp-staging.appspot.com/io2016/
2. Use the tab key to navigate to the video 
3. Tap enter to play video 
4. Note video does not play 

Chrome 48 Mac OS X 10.11 

![gio_206](https://cloud.githubusercontent.com/assets/7585604/13298526/e708eae0-daeb-11e5-8b3f-13e8708a70f2.png)
